src/components/Filters/Filters.tsx

```diff
@@ -212,6 +233,7 @@ class Filters extends React.Component<ComposedProps, State> {
         rightPopoverableActions={this.transformFilters(filters)}
```

Comment:
The `PopoverableAction` interface defined in `ConnectedFilterControl.tsx` also needs to be updated to extend `DisableableAction` instead of `Action`.

Reply:
The type of the `action` argument of the `activatorButtonFrom` method needs to be updated to `PopoverableAction` instead of `BaseAction`.

Reply:
Nice suggestion, this allows for disabling individual activators! Added it to the code!